Add feature to configure offset for total sum

seyfahni opened the pull request:
When starting to use timewarrior one might already have overtime, so I propose to add a setting that let's one define an offset that will be added to the sum.

At the moment the command output looks like this:
```
          date       actual       target        diff
    2024-07-15       8h:07m       8h:00m      0h:07m
           ...          ...          ...         ...
    2025-02-12       8h:01m       8h:00m      0h:01m
    2025-02-13       8h:56m       8h:00m      0h:56m
    2025-02-14       0h:57m       8h:00m     -7h:02m
     sub-total    1008h:54m    1024h:00m    -15h:05m
        offset       8h:51m       0h:00m      8h:51m
         total    1017h:45m    1024h:00m     -6h:14m
```

With the new option configured:
```
flextime.offset_total_by_minutes = 531
```

I'm open to suggestions on how to format / name the different parameters. Options I thought about:
* Put the offset at the very top
* Omit the sub-total
* Name the config option differently (e.g. `flextime.offset_minutes`, ...)

aibor commented:
Hi,

Thanks for the PR. I like the idea.

About the naming: I like `minutes_offset_total`. This is similar to `hours _per_day`.

For the output: From my point of view it makes sense to put the offset at the top. Just like a carry over from a previous period. The sub-total is not necessary then.

Does it work with negative values too? If so, would you mind adding a test for that as well?

On another note: I just changed the github actions config. Would you mind resyncing your fork and rebasing your PR to the current main branch? Hopefully the actions will start then.

seyfahni commented:
I've rebased the branch to the up-to-date master, moved the offset output, added the test and renamed the config value.

Should be good to go now and thanks to your fast reply.

I have a few other (but more complicated) features I'm thinking about, but not sure if you would consider them in-scope:
* #4
* #5

Let me know which of them you wouldn't mind getting a PR for, though I'll probably implement them all for myself.

aibor commented:
Thanks for the quick adjustments. Looks good to me.

I fixed my linter config and made some adjustments to fix the linter issues that came up with it in this PR.

I'll take a look at your proposals and let you know. Thanks for sharing your ideas. I appreciate it.
